sprocs/sync_assessments.sql

```diff
@@ -347,6 +380,85 @@ BEGIN
                         question_id = EXCLUDED.question_id
                     RETURNING aq.id INTO new_assessment_question_id;
                     new_assessment_question_ids := array_append(new_assessment_question_ids, new_assessment_question_id);
+
+                    IF (valid_assessment.data->>'group_work')::boolean THEN
+                        -- Create temporary tables for roles that can view the question
```

Comment:
Can you clarify why this works with temporary tables instead of just inserting directly into `assessment_question_role_permissions`?

Reply:
We realized that we don't need to use temporary tables, so we've updated the implementation to just insert directly into `assessment_question_role_permissions`.